test/WebJobs.Script.Tests/AggregateFunctionMetadataProviderTests.cs

```diff
@@ -1,326 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using System.IO;
-using System.IO.Abstractions.TestingHelpers;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.Script.Description;
-using Microsoft.Azure.WebJobs.Script.Diagnostics;
-using Microsoft.Azure.WebJobs.Script.Tests.Workers.Rpc;
-using Microsoft.Azure.WebJobs.Script.Workers;
-using Microsoft.Azure.WebJobs.Script.Workers.Rpc;
-using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Logging.Abstractions;
-using Microsoft.Extensions.Options;
-using Microsoft.WebJobs.Script.Tests;
-using Moq;
-using Xunit;
-
-namespace Microsoft.Azure.WebJobs.Script.Tests
-{
-    public class AggregateFunctionMetadataProviderTests
-    {
-        private readonly TestLogger _logger;
-        private AggregateFunctionMetadataProvider _aggregateFunctionMetadataProvider;
-        private Mock<IFunctionInvocationDispatcher> _mockRpcFunctionInvocationDispatcher;
-        private Mock<IFunctionMetadataProvider> _mockFunctionMetadataProvider;
-
-        public AggregateFunctionMetadataProviderTests()
-        {
-            _logger = new TestLogger("AggregateFunctionMetadataProviderTests");
-            _mockRpcFunctionInvocationDispatcher = new Mock<IFunctionInvocationDispatcher>();
-            _mockFunctionMetadataProvider = new Mock<IFunctionMetadataProvider>();
-        }
-
-        [Fact]
-        public void ValidateBindings_NoBindings_Throws()
-        {
-            FunctionMetadata functionMetadata = new FunctionMetadata();
-            List<string> rawBindings = new List<string>();
-
-            var ex = Assert.Throws<FormatException>(() =>
-            {
-                AggregateFunctionMetadataProvider.ValidateBindings(rawBindings, functionMetadata);
-            });
-
-            Assert.Equal("At least one binding must be declared.", ex.Message);
-        }
-
-        [Fact]
-        public void ValidateBindings_DuplicateBindingNames_Throws()
```

Comment:
Do we have plans to add these tests in a different class? It seems we are removing all these tests.